src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivClean.hpp

```diff
@@ -207,6 +208,10 @@ struct EvolutionMetavars {
   static_assert(
       is_analytic_data_v<initial_data> xor is_analytic_solution_v<initial_data>,
       "initial_data must be either an analytic_data or an analytic_solution");
+  // boolean verifying if supernova evolution.  If true, activate parameterized
+  // deleptonization
+  static constexpr bool evolve_ccsn =
```

Comment:
Maybe name this `include_parameterized_deleptonization`? That at least makes it very clear what it does. Up to you! You could even do:
```cpp
using parameterized_deleptonization = tmpl::conditional_t<...>;
```
and then just use `parameterized_deleptonization`. That would reduce overall code length & complexity IMO